### PR TITLE
Normalize regime transition probabilities

### PR DESCRIPTION
**Pull request by timmens**
In this PR, we add normalization of regime transition probabilities, which is necessary because otherwise agents assign probability to inactive regimes.

### CODE REVIEW

File: src/lcm/error_handling.py [hunk 1]
```diff
@@ -24,5 +24,9 @@ def validate_value_function_array(
     """
     if jnp.any(jnp.isnan(V_arr)):
         raise InvalidValueFunctionError(
-            f"The value function array in period {period} contains NaN values."
+            f"The value function array in period {period} contains NaN values. This "
+            "may be due to various reasons:\n"
+            "- The user defined functions returned invalid values.\n"
+            "- It is impossible to reach an active regime, resulting in NaN regime "
+            "transition probabilities in the normalized transition probabilities."
         )
```
Comment (hmgaudecker) [inline src/lcm/error_handling.py]:
```suggestion
            "- The user-defined functions returned invalid values.\n"
            "- It is impossible to reach an active regime, resulting in NaN regime\n"
            "  transition probabilities in the normalized transition probabilities."
        )
```

**Review by hmgaudecker — APPROVED** [review]
Would be great to test the exceptions, too. But no show-stopper!
